src/misc/he.json

```diff
@@ -0,0 +1,1002 @@
+{
+  "0": "טנץ', דג טנקה",
+  "1": "דג זהב,קָרַס זָהָב, Carassius auratus",
```

Comment:
please clean all English words from json file

Reply:
I've detailed in the first comment the reason for that. plus, in cases where the languages were combined was to give a more precise solution as described in the English labeling. is that clearer now?